### ISSUE TITLE
Retrieve search results.

### ISSUE THREAD

**Issue by pratikguru** [gregnb/mui-datatables#716]
Hey, I was wondering if there was a method to retrieve the search results after the callback has been triggered on search. I did not see ( or didn't look hard enough ) for a method or callback to retrieve the search results.

Has anyone else figured out a way to retrieve the search results?

**Comment by gabrielliwerant** [comment]
I don't believe there is a way to do this currently with the search callback, however, if you use `customSearch`, you can collect the results that you intend to appear from the search, so that will have the same effect. It's fairly simple to implement. There is an example in `customize-search` if you need some help.
